Program.cs

```diff
@@ -16,7 +19,16 @@ class Program {
       // Create a MandelWin that shows an animated Mandelbrot set,
       // and create an Application object to do message-pumping and keep
       // the window alive
+      Init ();
       new MandelWin ().Show ();
       new Application ().Run ();
    }
+
+   static void Init () {

```

Comment:
This is not good form. If you forget to call Init(), then DataDir is invalid. Why not place this code in the getter of DataDir, so it gets initialize the first time (if DataDir is "")